examples/proc_Ep.py

```diff
@@ -13,7 +13,7 @@
     thisfile, exptype, nodename, post_proc, lookup = (
         "240924_13p5mM_TEMPOL_ODNP_1.h5",
         "ODNP_NMR_comp/ODNP",
-        "ODNP",
+        "FIR_34dBm",
```

Comment:
You should be pulling an IR dataset -- not the ODNP dataset from the same file.

Reply:
This was ODNP, i think it just got changed in the wrong file